connectors/activemq/build.gradle

```diff
@@ -0,0 +1,9 @@
+dependencies {
+    api project(':forklift-core'),
+            'org.apache.activemq:activemq-client:5/14.0',
```

Comment:
Should this be `5.14.0`?

Reply:
Probably. I bet in came in transitively from plug-in. Going to look to see if I can remove it.

Reply:
I bet bringing in the broker brought in the client correctly. But fixing.